arviz/plots/dotplot.py

```diff
@@ -0,0 +1,142 @@
+import math
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dots(
+    values=None,
+    binwidth=None,
+    dotsize=1,
+    stackratio=1,
+    hdi_prob=None,
+    rotated=False,
+    dotcolor="grey",
+    intervalcolor="red",
+    markersize=None,
+    figsize=None,
+    linewidth=None,
+    point_estimate="auto",
+    quantiles=None,
```

Comment:
could the default value be a small integer like 20?

Reply:
I agree that it would probably be better to have a quantile dotplot as a default, especially if the dotplot with a lot of samples is slow.

Reply:
Ok, I can make quantile dot plot as default. Should I keep the default value as 20? or a bit more

Reply:
I agree quantile dotplot should be the default. I would check what [ggdist](https://mjskay.github.io/ggdist/) uses for the default number of dots and emulate that, since the author Matthew Kay does research in uncertainty visualizations.

Reply:
Sure